deployment.yml

```diff
@@ -137,7 +137,7 @@ spec:
           - name: HOST
             value: '10.3.0.12'
           - name: PORT
-            value: '8088'
+            value: '8081'
```

Comment:
Why does this port number need to be changed?

Reply:
Port number at 8088 seems to be blocking deployment, historically Ola has changed to 8081 from 80888 and put back the port.